packages/templates/clients/websocket/python/components/SendOperation.js

```
@@ -0,0 +1,48 @@
+  if (!sendOperations || sendOperations.length === 0) {
+    return null;
+  }
+
+  return (
+    <>
+      {
+        sendOperations.map((operation) => {
+          const staticMethodName = `${operation.id()}_static`;
+          
+          return (
+            <Text newLines={2} indent={2}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)